src/webgpu/api/operation/rendering/depth_bias.spec.ts

```diff
@@ -261,3 +339,40 @@ g.test('depth_bias')
   .fn(async t => {
     t.runDepthBiasTest('depth32float', t.params);
   });
+
+g.test('depth_bias_24bit_format')
+  .desc(
+    `
+  Tests that a square with different depth bias values like 'positive', 'negative', 'infinity',
+  'slope', 'clamp', etc. is drawn as expected with 24 bit depth format.
+  `
+  )
+  .params(u =>
+    u //
+      .combineWithParams([
+        {
+          quadAngle: QuadAngle.Flat,
+          bias: 0.25 * (1 << 25),
+          biasSlopeScale: 0,
+          biasClamp: 0,
+          expectedColor: new Float32Array([1.0, 0.0, 0.0, 1.0]),
+        },
+        {
+          quadAngle: QuadAngle.TiltedX,
+          bias: 0.25 * (1 << 25),
+          biasSlopeScale: 1,
```

Comment:
This case has the same expected result as the one above, so would still pass if biasSlopeScale were broken. See comment above.